RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  CustomerCenterView.swift
+//
+//
+//  Created by Andrés Boedo on 5/3/24.
+//
+
+import RevenueCat
+import SwiftUI
+
+@available(iOS 15.0, *)
+struct CustomerCenterView: View {
+
+    @StateObject private var viewModel = CustomerCenterViewModel()
+
+    init() { }
+
+    init(viewModel: CustomerCenterViewModel) {
+        self._viewModel = .init(wrappedValue: viewModel)
+    }
+
+    var body: some View {
+        NavigationView {
+            NavigationLink(destination: destinationView()) {
+                Text("Billing and subscription help")
+                    .padding()
+                    .background(Color.blue)
+                    .foregroundColor(.white)
+                    .cornerRadius(10)
+            }
+        }
+        .onAppear {
```

Comment:
Nit: use `.task` here instead of .onAppear and then you can make `checkAndLoadSubscriptions` async!

https://developer.apple.com/documentation/swiftui/view/task(priority:_:)